### PR TITLE
Use quote in env vars for node e2e tests

### PR DESCRIPTION
**Pull request by yguo0905**
This is to fix https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2enode-cosstable1-k8sstable1-default-defaultspec/1/

A local run from `docker run` seemed to work.

/assign @krzyzacy

### CODE REVIEW

File: experiment/test_config.yaml [hunk 1]
```diff
@@ -720,10 +720,10 @@ nodeTestSuites:
   default:
     envs:
     - TIMEOUT=60m
-    - GINKGO_FLAGS=--skip=\[Flaky\]|\[Serial\]
+    - GINKGO_FLAGS="--skip=\[Flaky\]|\[Serial\]"
   serial:
     - TIMEOUT=180m
-    - GINKGO_FLAGS=--focus=\[Serial\] --skip=\[Flaky\]|\[Benchmark\]
+    - GINKGO_FLAGS="--focus=\[Serial\] --skip=\[Flaky\]|\[Benchmark\]"
```
Comment (krzyzacy) [inline experiment/test_config.yaml]: please use `--test_args=--focus=foo --skip=bar` instead :-)

File: experiment/test_config.yaml [hunk 2]
```diff
@@ -690,7 +690,7 @@ nodeCommon:
     envs:
     - CLEANUP=true
     - GCE_ZONE=us-central1-f
-    - KUBELET_ARGS=--cgroups-per-qos=true --cgroup-root=/
+    - KUBELET_ARGS="--cgroups-per-qos=true --cgroup-root=/"
```
Comment (krzyzacy) [inline experiment/test_config.yaml]: env file should not contain quotes (you can look at other env files, and we have unit-test guard all e2e env files does not have quotes), I think instead we need to fix the place references it

Comment (yguo0905) [inline experiment/test_config.yaml]: `kubernetes_kubelet.py` scenario does not support passing `--test_args` and its usage of the envs requires quotes. I planed to have the test running first and then migrate those envs to flags, as what we've been doing for e2e scenario. That's why I temporarily make an exception for node-e2e tests in `bootstrap_test.py` in this PR. But I think I can also do this now.

Comment (krzyzacy) [inline experiment/test_config.yaml]: If you will send follow up PRs to fix the TODO and you want to see the test result then I'm ok to merge this first :-)

Comment (yguo0905) [inline experiment/test_config.yaml]: That will be cool. :) I will send follow up PRs later.

**Comment by krzyzacy** [comment]
Opened an issue to track this :-) I'll get this in

**Pull request merged**